### ISSUE TITLE
pymc.utils.coda() problem

### ISSUE THREAD

**Issue by thommevans** [pymc-devs/pymc3#118]
I'm wondering if there's a minor bug in the coda_output() function definition in the utils module. Specifically, the exception handling at around line 725:

```
    try:
        index = _process_trace(trace_file, index_file, v.trace(), vname, index)
    except TypeError:
        pass
```

doesn't seem to work always. For instance, here's a simple example with output where I get an AttributeError rather than a TypeError, so it crashes: 

In [2]: from pymc.examples import disaster_model

In [3]: M=pymc.MCMC(disaster_model)

In [4]: M.sample(iter=1000)
[***************_100%**_***************]  1000 of 1000 complete

In [5]: pymc.utils.coda(M)
# Generating CODA output

Processing rate
Processing early_mean
Processing late_mean
## Processing disasters

AttributeError                            Traceback (most recent call last)
/Users/tomevans/<ipython-input-5-d2ff7d65d837> in <module>()
----> 1 pymc.utils.coda(M)

/Users/tomevans/.local/lib/python2.6/site-packages/pymc/utils.py in coda_output(pymc_object)
    723         print_("Processing", vname)
    724         try:
--> 725             index = _process_trace(trace_file, index_file, v.trace(), vname, index)
    726         except TypeError:
    727             pass

AttributeError: 'Poisson' object has no attribute 'trace'

Or maybe I'm just not using the function quite as it's designed (I tried following the available documentation)...

**Comment by fonnesbeck** [comment]
Thanks for reporting this. Should be fixed in 3966c746c7a56a8c3095e64a1e7605779bd883c8. Please test.

**Comment by thommevans** [comment]
Thank you, that seems to be all that was needed. The above example I posted now works as expected.
